lib/src/token.dart

```diff
@@ -93,7 +93,7 @@ class TagToken implements Token {
   final FileSpan span;
 
   /// The tag handle for named tags.
-  final String handle;
+  final String? handle;
 
   /// The tag suffix, or `null`.
   final String suffix;
```

Comment:
The comment indicates that this can be `null`. If that's not true (and the type suggests that it isn't) we should update the comment. But if it's expected to be `null` and the migration engine mis-migrated it, then we should make it nullable.

Reply:
Good catch.  Thanks!